chaosk8s/node/actions.py

```diff
@@ -238,7 +238,6 @@ def drain_nodes(name: str = None, label_selector: str = None,
     for node in nodes:
         node_name = node.metadata.name
         ret = v1.list_pod_for_all_namespaces(
-            include_uninitialized=True,
```

Comment:
Any reason for this change?

Maybe the API has changed?

Reply:
Yes, this parameter does not seem to exist on the version of the API that we use.

Reply:
Indeed, I couldn't find it either. It's odd it was there.